src/combat.cpp

```diff
@@ -943,7 +943,7 @@ void ValueCallback::getMinMaxValues(Player* player, CombatDamage& damage, bool u
 		}
 
 		case COMBAT_FORMULA_SKILL: {
-			//onGetPlayerMinMaxValues(player, attackSkill, attackValue, attackFactor)
+			//onGetFormulaValues(player, level, attackSkill, attackValue, attackFactor)
```

Comment:
Because it no longer puts the levelPower instead of level?

player-> GetLevel () / 5?